config.js

```diff
@@ -10,6 +10,7 @@ let config = {
     mongo_db_password: process.env.MONGO_DB_PASSWORD,
     mongo_db_host: process.env.MONGO_DB_HOST,
     mongo_db_port: process.env.MONGO_DB_PORT,
+    official_email: process.env.official ? process.env.OFFCIAL_EMAIL : 'CRDCHelpDesk@nih.gov',
```

Comment:
There are two different environment variables in this initialization line. I'm assuming that process.env.official and process.env.OFFICIAL_EMAIL should be the same variable. 

we can use the || operator here instead to do the same thing and avoid potential issues like this in the future:
`official_email: process.env.OFFICIAL_EMAIL || 'CRDCHelpDesk@nih.gov',`

Reply:
updated